check_pslse.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+#
+# Copyright 2017 International Business Machines
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+###############################################################################
+cd ${PSLSE_ROOT}
```

Comment:
maybe an if [ -z ${PSLSE_ROOT} ]; ... is nice to avoid trouble if the env variable is not yet set properly.
Alternatively to the env vars parameters might be nice, but that is probably just a taste issue.